lib/notifier.es

```diff
@@ -31,6 +31,7 @@ export class Notifier {
         this._loader.getSubtitles().then((data) => {
             this._subtitles.ships = data;
             this.__ = I18nService.getPluginI18n();
+            this.__r = (x) => window.i18n.resources.fixedT(x, {lng: I18nService.getLocale()});
```

Comment:
If `poi-plugin-translator` is not installed, `window.i18n.resources.fixedT` will not be available and cause crash.